xmppserver/src/main/java/org/jivesoftware/openfire/PresenceManager.java

```diff
@@ -66,7 +66,7 @@ public interface PresenceManager {
      * Returns all presences for the user, or {@code null} if the user is unavailable.
      *
      * @param username the name of the user.
-     * @return the Presence packets for all the users's connected sessions.
```

Comment:
Actually this should be " users' ". We made this extra complicated to addle non-native speakers' minds.

Reply:
Yeah, they're all singular, forget I said anything.

Reply:
I actually checked that one. :)